### PR TITLE
phpstan updated to 0.11.3

### PR DESCRIPTION
**Pull request by adaamz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -54,8 +54,8 @@ jobs:
       env: COVERAGE=yes
 
     - stage: Lint
-      install: travis_retry composer require --dev --prefer-dist phpstan/phpstan:^0.7
-      script: vendor/bin/phpstan analyse -l 3 -c phpstan.neon lib tests
+      install: travis_retry composer require --dev --prefer-dist phpstan/phpstan:^0.11 phpstan/phpstan-phpunit
+      script: vendor/bin/phpstan analyse -l 5 -c phpstan.neon lib tests
```
Comment (SenseException) [inline .travis.yml]: Can you please move the level and paths to a phpstan.neon.dist file? Then we should be able to remove phpstan.neon and put it into the gitignore file.

**Review by SenseException — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (SenseException) [reply .travis.yml]: Can you also please add phpstan to the dev dependencies of composer?

Comment (CZechBoY) [reply .travis.yml]: I also commited `composer.lock` as is standard in other `doctrine/*` projects.

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
Thanks!

**Comment by Ocramius** [comment]
@CZechBoY can I ask you to check this again? I realised that it failed post-merge, sorry :-(

**Comment by greg0ire** [comment]
There are not many issues, I'm on it :)

**Comment by CZechBoY** [comment]
@Ocramius I think something was merged before this :-/ Always rebase merge strategy to the win! :D

**Pull request merged**